WalletWasabi/Rpc/JsonRpcResponse.cs

```diff
@@ -26,10 +26,10 @@ private JsonRpcResponse(string id, object result, object error)
 	public string JsonRpc => "2.0";
 
 	[JsonProperty("result", Order = 1)]
-	public object Result { get; }
+	public object? Result { get; }
 
 	[JsonProperty("error", Order = 1)]
-	public object Error { get; }
+	public object? Error { get; }
```

Comment:
Both of them have `Order = 1`, looks like a leftover mistake. Can you fix it please?

Reply:
I think it's not a mistake, this response can either be a result or an error, in either case the other property is null. There is no response where neither `Result` and `Error` isn't null.

Reply:
Oops, yes, you are right. Nevermind then.